{{cookiecutter.project_slug}}/backend/app/app/schemas/user.py

```diff
@@ -1,40 +1,49 @@
 from typing import List, Optional
+from datetime import datetime
 
-from pydantic import BaseModel
+from pydantic import BaseModel, Schema
 
 
 # Shared properties
 class UserBase(BaseModel):
-    email: Optional[str] = None
-    full_name: Optional[str] = None
-    is_active: Optional[bool] = True
-    is_superuser: Optional[bool] = False
+    email:str = None
```

Comment:
Not so sure about removing all the `Optional`... Technically, a `str` type cannot have `None` as the value, right? Plus this does not work well with mypy.

Reply:
Hi @HymanZHAN ,

I had some issues with the Optionnal[] on native types (int, bool, str, ...), but I am not able to remember which ones exactly... but that is the reason why I removed them.

`None` should not be considered as the value of the string here, but more an indication that it is not set (i.e optionnal) but I agree with you that it is not as pretty as the original Optionnal[]

I will have to digg into it for the source pb :sweat_drops: